slides/fosdem14/index.html

```diff
@@ -20,6 +21,7 @@
 <section>
   <section data-markdown>
     <script type="text/template">
+  <!-- <img width="150" src="../ucnlogo/ucn_logo.png" style="margin-left: 945px; margin-top: 160px; position: fixed" /> -->
```

Comment:
please remove any extraneous lines, eg., this one is commented out so just delete it.